modules/ocf_backups/files/upload-to-box

```diff
@@ -164,6 +105,26 @@ def get_access_token(creds):
         return response['access_token']
 
 
+def upload_to_box(creds):
+    """Uploads the archive to box at the configured location"""
+    lftp_cmd = '''
+        set ftps:initial-prot ""
+        set ftp:ssl-force true
+        set ftp:ssl-protect-data true
+        open {box_ftp}
+        user ocf@berkeley.edu {passwd}
```

Comment:
I think this email should come from `creds['email']` instead of being hardcoded here (or it should be removed from the private share's creds in favor of having it be a constant in this script).